rvv-intrinsic-generator/rvv_intrinsic_gen/generator.py

```diff
@@ -278,11 +281,12 @@ class APITestGenerator(Generator):
   Derived generator for api unit tests.
   """
 
-  def __init__(self, f, is_overloaded, is_llvm, has_tail_policy):
+  def __init__(self, f, is_overloaded, is_llvm, is_gnu, has_tail_policy):
```

Comment:
nit: we can fuse `is_llvm, is_gnu` to a enum I guess.

Reply:
how about `llvm` and `gnu` ? As `is_xx` looks like not a common used name for enum.